src/index.js

```diff
@@ -43,6 +43,13 @@ export default function eslint(options = {}) {
                 err.name = 'ESLintError';
                 throw err;
             }
+
+            if (options.breakOnWarning) {
+                const err = Error(' ');
+                err.name = ' ';
```

Comment:
@TrySound my question would be, do we need this new option, or we just remove the stack from the error we throw in case of `throwError`